src/platform_impl/apple/appkit/view.rs

```diff
@@ -15,6 +15,11 @@ use objc2_foundation::{
     NSArray, NSAttributedString, NSAttributedStringKey, NSCopying, NSMutableAttributedString,
     NSNotFound, NSObject, NSPoint, NSRange, NSRect, NSSize, NSString, NSUInteger,
 };
+use winit_core::event::{
```

Comment:
I think It'd make sense to do just the `crate::` -> `winit_core::` moving in a separate PR, to make things easier to review.

Could be done with a `extern crate self as winit_core;` in `lib.rs`.

Reply:
The thing is that it's just all like that, there's no actual changes other than those renames in this PR.

Reply:
> there's no actual changes other than those renames in this PR.

Respectfully, I don't think that's true. You've commented code out, added a `build.rs` file, changed the visibility of some fields, etc.

Reply:
it's a draft for a reason. Please don't review un-finished code.